tools/downloader/pytorch_to_onnx.py

```diff
@@ -8,19 +8,29 @@
 import torch
 import torch.onnx
 
+import numpy as np
+import ast
+
 
 def positive_int_arg(values):
     """Check positive integer type for input argument"""
+    inputs = ast.literal_eval(values)
+    number_of_inputs = np.ndim(inputs)
```

Comment:
This is wrong. First of all, `ndim` does not give you the number of inputs, but the nesting level. `ndim([[1], [2], [3]])` is 2, yet that list corresponds to 3 inputs.

But more importantly, `ndim` won't even give you the expected result in some cases. `ndim([[1,2,3], [1,2]])` is 1.

There's no need to involve NumPy at all here. What you need to do is:

* verify that `inputs` is a sequence;
* if all elements of `inputs` are integers, wrap it in a tuple;
* for each element of `inputs`:
   * verify it's a sequence;
   * verify that every subelement is a positive integer.

You don't need a `result` variable, because the function can only return one value, `inputs`.